scripts/zones/Lower_Jeuno/npcs/Treasure_Coffer.lua

```diff
@@ -478,6 +590,77 @@ local function givePrize(player, ki)
     end
 end
 
+
+function addonGear(player, option, addonReward)
```

Comment:
From CI:
`    scripts/zones/Lower_Jeuno/npcs/Treasure_Coffer.lua:594:10: (W111) setting non-standard global variable addonGear`

Please mark your local helper functions as `local`, otherwise they default to being global.

https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md#local-vars-are-almost-always-preferred